website/docs/02_getting-started/coding.mdx

```diff
@@ -616,4 +616,4 @@ You can find the code both for the frontend and the backend in <CLAskMeRepo>[thi
 
 ### All the guides and examples
 
-Check out the <CLStepByStep>[step-by-step guides](https://github.com/boostercloud/booster/tree/main/examples)</CLStepByStep> and the <CLExampleApps>[example apps repository](https://github.com/boostercloud/examples)</CLExampleApps> to see Booster in use.
+Check out the <CLStepByStep>[step-by-step guides](https://github.com/boostercloud/examples)</CLStepByStep> and the <CLExampleApps>[example apps repository](https://github.com/boostercloud/examples)</CLExampleApps> to see Booster in use.
```

Comment:
Let's remove the redundant link before merging, I'd have done the change myself, but I can't push to your branch. Here you have a one-click change. Once it's applied, we can merge it.

```suggestion
Check out the <CLExampleApps>[example apps repository](https://github.com/boostercloud/examples)</CLExampleApps> to see Booster in use.
```